backend/hexa/user_management/utils.py

```diff
@@ -98,7 +78,7 @@ def send_organization_add_user_email(
     from django.utils.translation import gettext_lazy, override
 
     from hexa.analytics.api import track
-    from hexa.core.utils import send_mail
+    from hexa.core.utils import get_email_attachments, send_mail
```

Comment:
[Detail] Is there a reason why the imports are at the method level in `send_organization_invite` and `send_organization_add_user_email`?